release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -964,6 +964,19 @@ See xref:../release_notes/versioning-policy.adoc#ocp-versioning-policy[OpenShift
 
 See link:https://kubernetes.io/docs/setup/release/version-skew-policy/[Kubernetes documentation] for details about version skew policies in the upstream Kubernetes project.
 
+[discrete]
+[id="ocp-4-7-pod-topologys-spread-constraints"]
+==== Scheduler now uses pod topology spread constraints
+
+The default scheduler in {product-title} 4.7 now uses pod topology spread constraints to control the placement of pods. Ensure that your nodes have the required labels in order for pod replicas to spread properly.
+
+[NOTE]
+====
+By default, the scheduler requires the `kubernetes.io/hostname` and `topology.kubernetes.io/zone` node labels. If your nodes will not use these labels, then define your own pod topology spread constraints instead of using the default constraints.
```

Comment:
Not sure if "then" is necessary here. ISG recommends against this construction unless it improves clarity where you are describing a causative relationship.

```suggestion
By default, the scheduler requires the `kubernetes.io/hostname` and `topology.kubernetes.io/zone` node labels. If your nodes will not use these labels, define your own pod topology spread constraints instead of using the default constraints.
```